### PR TITLE
Fixes #20035 - Adds API link to About page

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: app/views/about/index.html.erb [hunk 1]
```diff
@@ -124,7 +124,8 @@
       <h4><%= _("Support") %></h4>
       <h6><%= _("Documentation") %></h6>
       <ul>
-        <li><%= link_to _('Manual'),documentation_url, :rel => "external" %></li>
+        <li><%= link_to _('API'), apipie_apipie_path %></li>
```
Comment (ohadlevy) [inline app/views/about/index.html.erb]: nitpick: should this be API docs or something more clear?

Comment (ares) [reply app/views/about/index.html.erb]: Yeah... I was not sure about the label, "API documentation" seems like too much for just a list of endpoints. How about "Available API calls" or "API provided by the server" or simply "Server API"?

Comment (ohadlevy) [reply app/views/about/index.html.erb]: I would add it maybe via an hover or an icon? something like
```Automate Foreman via a simple and powerful API```

**Review by dLobatog — CHANGES_REQUESTED** [review]
Setting to WoC as per @ohadlevy 's comment. @ares  Personally I think "API documentation" or "API resources"  is accurate to describe what Apipie provides.

**Comment by ares** [comment]
I liked "API Resources" the most, I also added the hover message as @ohadlevy suggested

**Comment by dLobatog** [comment]
Failure http://ci.theforeman.org/job/test_develop_pr_core/13327/database=postgresql,ruby=2.4,slave=fast/testReport/junit/(root)/ComputeResourceJSIntegrationTest/test_0003_compute_resource_password_isn_t_deleted_when_testing_connection/ should be fixed by https://github.com/theforeman/foreman/pull/4604

**Review by dLobatog — APPROVED** [review]

**Comment by dLobatog** [comment]
Thanks @ares !

**Pull request merged**